src/utils/customFetch.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpVerbs } from "src/@types/HttpVerbs";
+
+export function customFetch(
+  endpoint: string,
+  method: HttpVerbs,
```

Comment:
We should restrict that to the methods that are not `GET` since we won't be able to send a body with a GET

Reply:
Hamza, you could try to use [utility types](https://www.typescriptlang.org/docs/handbook/utility-types.html) to keep the enum (i.e. `Exclude`). I never used them on enums, but I don't see a reason why it wouldn't work.

Reply:
OK fixed!

Reply:
One trick you can do, is copy the commit id, and post it in a comment, like this a8f2ae1.